CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+
+### v3.4.2
+
+**Fixed**
+- Fix clair crash downloading RHEL content mapping
+- Quay config validates SMTP
+- Quay Operator mirror configurations cannot be decoded after QuayRegistry
```

Comment:
Can you rephrase this. I'm not sure what changed from reading this line.

Reply:
@alecmerdler do you have better wording?

Reply:
"Mirror configurations cannot be decoded when using Quay Operator"